lib/lightning/impact_tracking/report.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Lightning.ImpactTracking.Report do
+  @moduledoc """
+  Report submitted to ImpactTracker
+
+  """
+  use Ecto.Schema
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  schema "impact_tracking_reports" do
+    field :data, :map
```

Comment:
I imagine you've picked a generic term as it probably will save other kind of data besides metrics, right?

Reply:
Yes - the idea is that `data` will contain everything that is sent to the ImpactTracker - so the metrics as well as any metadata that may be required (e.g. the time the metrics were generated)